ui/components/ui/form-combo-field/form-combo-field.tsx

```diff
@@ -82,18 +82,15 @@ function Option({
 
 function Dropdown({
   maxDropdownHeight,
-  noOptionsText,
   onOptionClick,
   options,
   width,
 }: {
   maxDropdownHeight?: number;
-  noOptionsText?: string;
```

Comment:
Since the goal of this was to provider a reusable form combo field decoupled from petnames, could we instead add an optional boolean property argument to hide the dropdown if there are no options?

Something like `hideDropdownIfNoOptions`?

Reply:
I thought we wouldn't want to keep unused code in the base since only use-case for this is petnames. 
But regardless I put them back in https://github.com/MetaMask/metamask-extension/pull/22466/commits/e5c696a87dbcff6c5429b2d791c9ea4c54780371

Please not that I drilled down the prop to avoid covering more logic to set visibility in a `useEffect`

Reply:
I thought we wouldn't want to keep unused code since the only use-case here is petnames. 

[This commit](https://github.com/MetaMask/metamask-extension/commit/38d5f3db4ce8c85a56012bd49d8b3c383dcc280b) puts it back and "noOptions" case and `hideDropdownIfNoOptions`

Please note that I drilled down `hideDropdownIfNoOptions` to `Dropdown` component to avoid putting more logic to set visibility in a `useEffect`